tst/broken/quick/ProjSubfield3.tst

```diff
@@ -1,10 +1,8 @@
+# This test breaks if you run it e.g. 50 times in a row.
```

Comment:
Hmm, I tried this, and yes it failed: but the issue was that it unexpectedly said:
```
#I  Giving up, Schreier tree is not shallow.
```

This message is from `orb`, and can be disabled via `SetInfoLevel(InfoOrb, 0);`; so we could just do that before the last line of this test (and then restore the old info level afterwards)

Reply:
Done. I'm now gonna finish the actions PR.